yodeploy/tests/test_util.py

```diff
@@ -77,18 +78,18 @@ def test_permission_squash(self):
             'PATH': os.environ['PATH'],
             'PYTHONPATH': ':'.join(sys.path),
         }
-        subprocess.check_call((
-            'fakeroot',
-            'python',
-            '-c', (
-                'import yodeploy.util, os; '
-                'yodeploy.util.extract_tar("%s", "%s"); '
-                's = os.stat("%s"); '
-                'assert s.st_uid == 0; '
-                'assert s.st_gid == 0'
-            ) % (self.tmppath('test.tar.gz'), self.tmppath('extracted'),
-                 self.tmppath('extracted/bar'))
-        ), env=env)
+        test_script = (
+            'import yodeploy.util, os; '
+            'yodeploy.util.extract_tar("%s", "%s"); '
+            's = os.stat("%s"); '
+            'assert s.st_uid == 0; '
+            'assert s.st_gid == 0'
+        ) % (
+            self.tmppath('test.tar.gz'), self.tmppath('extracted'),
+            self.tmppath('extracted/bar'))
+
+        subprocess.check_call(
+            ('fakeroot', sys.executable, '-c', "'%s'" % test_script), env=env)
```

Comment:
What's with the single quotes around the script? There shouldn't be any shells interpreting this.